web/src/beta/features/Editor/Tabs/Storytelling/StoryPanel/index.tsx

```diff
@@ -1,18 +1,33 @@
 import { FC } from "react";
 
+import StoryPageIndicator from "@reearth/beta/features/Editor/Tabs/Storytelling/StoryPageIndicator";
 import { styled } from "@reearth/services/theme";
 
 type Props = {};
 
-const StoryPanel: FC<Props> = () => {
-  return <Wrapper>StoryPanel</Wrapper>;
+export const StoryPanel: FC<Props> = () => {
+  return (
+    <Wrapper>
+      <StoryPageIndicator
+        currentPage={3}
+        currentPageProgress={33}
+        maxPage={6}
+        onChangePage={page => console.log(page)}
```

Comment:
Do we want to keep this log statement?

Reply:
yes, I think this is no problem since definitely replace by the end of storytelling task.